prowler/providers/aws/services/documentdb/documentdb_cluster_deletion_protection/documentdb_cluster_deletion_protection.metadata.json

```diff
@@ -8,7 +8,7 @@
   "ResourceIdTemplate": "arn:aws:rds:region:account-id:db-cluster",
   "Severity": "medium",
   "ResourceType": "AwsRdsDbClusters",
-  "Description": "Check if Neptune Clusters has deletion protection enabled.",
+  "Description": "Check if DocumentDB Clusters has deletion protection enabled.",
   "Risk": "Enabling cluster deletion protection offers an additional layer of protection against accidental database deletion or deletion by an unauthorized user. A Neptune DB cluster can't be deleted while deletion protection is enabled. You must first disable deletion protection before a delete request can succeed.",
```

Comment:
There are more references to Neptune here.

Reply:
As jfagoagas mentioned, there is also an incorrect reference to "Neptune" in the Risk field that should be updated to "DocumentDB."

Reply:
True, thanks